MyComics/Flows/DetailCharacter/View/DetailCharacterViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  CharacterDetailView.swift
+//  MyComics
+//
+//  Created by Andrey Piskunov on 10.07.2023.
+//
+
+import UIKit
+
+final class DetailCharacterViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    private var presenter: DetailCharacterViewOutput
+    
+    private lazy var image = UIImage(named: "characterImage")
+    private lazy var characterImageView = UIImageView(image: image)
+    
+    private lazy var nameCharacterLabel: UILabel = {
+        let nameCharacterLabel = UILabel()
+        nameCharacterLabel.text = "Name"
+        nameCharacterLabel.textColor = .white
+        nameCharacterLabel.textAlignment = .center
+        return nameCharacterLabel
+    }()
+    
+    private lazy var descriptionLabel: UILabel = {
+        let descriptionLabel = UILabel()
+        descriptionLabel.text = "Герой американских комиксов Marvel Comics, обладающий сверхспособностями: силой, ловкостью, легкость"
+        descriptionLabel.textColor = .white
+        descriptionLabel.textAlignment = .center
+        descriptionLabel.numberOfLines = 10
+        return descriptionLabel
+    }()
+    
+    // MARK: - Init
+    
+    init(presenter: DetailCharacterViewOutput) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        presenter.viewIsReady()
+    }
+    
+    // MARK: - Private methods
+    
+    private func configureUI() {
+        view.backgroundColor = .commonBackground
```

Comment:
```suggestion
        navigationItem.largeTitleDisplayMode = .never
        view.backgroundColor = .commonBackground
```

поправил баг навигейшен бара

Reply:
поправил баг навигейшена